tests/datasets/test_profiles_processor.py

```diff
@@ -41,13 +92,57 @@ def serialize(self) -> Mapping[str, Any]:
 
     def build_result(self, meta: KafkaMessageMetadata) -> Mapping[str, Any]:
         result = asdict(self)
+        result["profile"] = ""
         result["received"] = datetime.utcfromtimestamp(self.received)
         result["offset"] = meta.offset
         result["partition"] = meta.partition
         return result
 
 
 class TestProfilesProcessor:
+    def test_sample_profile_message(self) -> None:
```

Comment:
this is only testing messages with a version field. What if there is no version field?

Reply:
A message with no version field would be the current message we process and it's tested here: https://github.com/getsentry/snuba/pull/3173/files#diff-4ced175fc87fe811e95d6e65609911fb917d0c2389691f91190968e9f531076fR181.